### ISSUE TITLE
Sega Mega Drive Classics. CUSA09771

### ISSUE THREAD

**Issue by and468860** [red-prig/fpps4-game-compatibility#374]
Message: Access violation (216)
  $0000000005011264 offset $00021264 libc.prx:abort+$00000004
  $0000000801096B53 offset $01096B53 eboot.bin:Entry+$01096AD3
  $00000000050120AA offset $000220AA libc.prx:realloc+$0000000A
  $000000000C9582DD offset $001182DD mono-ps4.prx:getenv+$000002ED
  $000000000C863E6B offset $00023E6B mono-ps4.prx:mono_jit_info_table_find+$000013AB
  $000000000C8FB1EF offset $000BB1EF mono-ps4.prx:Unknow+$0000096F
  $000000000C84802A offset $0000802A mono-ps4.prx:mono_assembly_load_from_full+$0000046A
  $000000000C847B21 offset $00007B21 mono-ps4.prx:mono_assembly_open_full+$00000381
  $000000000C866280 offset $00026280 mono-ps4.prx:mono_domain_assembly_open+$00000250
  $0000000800A5C221 offset $00A5C221 eboot.bin:Entry+$00A5C1A1
  $0000000800A59FA4 offset $00A59FA4 eboot.bin:Entry+$00A59F24
  $0000000800ABB94C offset $00ABB94C eboot.bin:Entry+$00ABB8CC
  $0000000800AC12ED offset $00AC12ED eboot.bin:Entry+$00AC126D
  $0000000800AC48F0 offset $00AC48F0 eboot.bin:Entry+$00AC4870
  $000000080108DA46 offset $0108DA46 eboot.bin:Entry+$0108D9C6
  $0000000801095099 offset $01095099 eboot.bin:Entry+$01095019
  $00000008000000BF offset $000000BF eboot.bin:Entry+$0000003F
  $0000000000000001 fpPS4.exe

**Comment by gandalfthewhite19890404** [comment]
Retest, as https://github.com/red-prig/fpPS4/commit/df6317ba0981983b2f9941034993ecbbe394c4bf was committed

**Comment by and468860** [comment]
![Uploading SEGA.Mega.Drive.classics.jpg…]()

**Comment by Kalorte** [comment]
https://github.com/red-prig/fpPS4/commit/07d736c7bcb927b6de45492106185d90135423bb

Same for me (Menu)
I found this on the log (in red):
nop nid:libSceHmd:B3F27AE9AAFD839D:sceHmdInitialize315

**Comment by and468860** [comment]
SEGA.Mega.Drive.Classics
Message: Access violation (216)
  $00000008006854C1 offset $006854C1 eboot.bin:Entry+$00685441
  $0000000801088519 offset $01088519 eboot.bin:Entry+$01088499
  $0000000801095099 offset $01095099 eboot.bin:Entry+$01095019
  $00000008000000BF offset $000000BF eboot.bin:Entry+$0000003F
  $0000000000000001 fpPS4.exe

**Comment by ArbestRi** [comment]
Retest with latest build

**Comment by ArbestRi** [comment]
`[main:31596] nop nid:libSceVoiceQoS:96F342961347CF12:sceVoiceQoSCreateLocalEndpoint`

**Comment by Hermiten** [comment]
![Capture d’écran 2023-04-01 170123](https://user-images.githubusercontent.com/151466/229296998-72c92b12-7b34-433e-9033-39333cb5f036.png)
![Capture d’écran 2023-04-01 170148](https://user-images.githubusercontent.com/151466/229297002-055fcbf2-3d05-42ab-8ca7-68f6b7200928.png)

It goes further but stuck at a black screen like thirst picture
And then I'm spammed by this
"sceNetRecvfrom:0"
So I think something wrong with the last commit about the sceNet...

**Comment by Hermiten** [comment]
> `[main:31596] nop nid:libSceVoiceQoS:96F342961347CF12:sceVoiceQoSCreateLocalEndpoint`

Hmm, Why don't you have the same result as me?

**Comment by gs-soda** [comment]
I'm not too sure, but sceNetRecvfrom was implemented yesterday, did you add the .prx file?

**Comment by Kimieggie** [comment]
@gs-soda sceNetRecvfrom is not a whitelisted lib

**Comment by Hermiten** [comment]
No, it's a fresh install, there is no additional lib on my folder

**Comment by gs-soda** [comment]
> > `[main:31596] nop nid:libSceVoiceQoS:96F342961347CF12:sceVoiceQoSCreateLocalEndpoint`
> 
> Hmm, Why don't you have the same result as me?

Just tested the game, the crash message appears just before the sceNetRecvfrom spam.

**Comment by GamingDucking** [comment]
Retest

**Comment by KITT1995** [comment]
An unhandled exception occurred at $0000000100296A99:
EAssertionFailed: SOPK?3 (emit_sopk.pas, line 64)
  $0000000100296A99
  $00000001002B1427
  $00000001002B14F1
  $00000001001406DC
  $0000000100140B49
  $0000000100140D61
  $00000001000935F4
  $000000010009457D
  $0000000100094AF0
  $00000001000909FC
  $0000000100005176
  $0000000100014611
  $00007FFB0B408D17
  $00007FFB0CD87450

XInput quit!
An unhandled exception occurred at $000000010023CCEB:
EAccessViolation: Access violation

**Comment by GamingDucking** [comment]
Retest

**Comment by KITT1995** [comment]
Keep spamming:
[FMOD mixer thread:17264] pthread_attr_init
[FMOD mixer thread:17264] pthread_attr_destroy

**Comment by gandalfthewhite19890404** [comment]
Still boots to menus?

**Comment by KITT1995** [comment]
Yes boots to menu
